### PR TITLE
Update dependencies for Debian stable/testing

### PR DESCRIPTION
**Pull request by waterkip**
I've also updated the README.md to reflect some changes for Ubuntu 18.04
as these are now the same as for Debian testing.

Closes: (GH #415)

### CODE REVIEW

**Comment by waterkip** [comment]
I've added and pushed my sign-off in the commit message (it isn't visible on github itself).

File: README.md [hunk 1]
```diff
@@ -54,39 +54,44 @@ sudo yum install openssl libcurl libxml2 pinentry xclip openssl-devel libxml2-de
...
+  libssl1.0-dev \
+  libxml2 \
   libxml2-dev  \
-  openssl  \
-  pinentry-curses \
-  pkg-config  \
-  xclip
+  pkg-config \
+  ca-certificates \
+  xclip \
```
Comment (rutkai) [inline README.md]: Trailing slash is not needed in the last line.

File: README.md [hunk 2]
```diff
@@ -54,39 +54,44 @@ sudo yum install openssl libcurl libxml2 pinentry xclip openssl-devel libxml2-de
...
   libxml2 \
-  libxml2-dev \
-  openssl \
-  pinentry-curses \
+  libxml2-dev  \
+  libssl1.1 \
   pkg-config \
-  xclip
+  ca-certificates \
+  xclip \
```
Comment (rutkai) [inline README.md]: Same here

File: contrib/Dockerfile [hunk 3]
```diff
@@ -1,4 +1,5 @@
-FROM debian:testing-slim
+FROM debian:testing-slim as build
+#FROM ubuntu:18.04
```
Comment (rutkai) [inline contrib/Dockerfile]: Please remove comment

File: contrib/Dockerfile.dev [hunk 4]
```diff
@@ -7,20 +7,21 @@ WORKDIR /tmp/build
...
 		pkg-config \
+		# Run time dependencies
+		libssl1.1 \
+		libcurl4  \
+		libxml2 \
+		# Optionals handy for testing within the container
+		bash-completion \
 		ca-certificates \
-		xclip
+		xclip \
```
Comment (rutkai) [inline contrib/Dockerfile.dev]: No trailing slash here either

File: README.md (hunk 1 above)
Comment (waterkip) [reply README.md]: Fixed.

File: README.md (hunk 2 above)
Comment (waterkip) [reply README.md]: Fixed

File: contrib/Dockerfile (hunk 3 above)
Comment (waterkip) [reply contrib/Dockerfile]: And done

File: contrib/Dockerfile.dev (hunk 4 above)
Comment (waterkip) [reply contrib/Dockerfile.dev]: dito

**Comment by rutkai** [comment]
Thank you! ;)

**Pull request merged**